File: 11650/11650.cpp
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   11650.cpp                                          :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2022/03/02 23:01:39 by jihoolee          #+#    #+#             */
/*   Updated: 2022/03/02 23:20:42 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

#include <iostream>
#include <vector>
#include <algorithm>
#include <utility>

int main(void) {
  int len;
  std::cin >> len;
  std::vector<std::pair<int, int> > vec(len);
  int first, second;

  for (int i = 0; i < len; ++i) {
    std::cin >> first >> second;
    vec[i] = std::pair<int, int>(first, second);
  }
  std::sort(vec.begin(), vec.end());
  for (int i = 0; i < len; ++i) {
    std::cout << vec[i].first << " " << vec[i].second << '\n';
  }
  return 0;
}

File: 11650/11650.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    11650.py                                           :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2022/03/02 23:22:53 by jihoolee          #+#    #+#              #
#    Updated: 2022/03/02 23:29:09 by jihoolee         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

length = int(input())
coords = [list(map(int, input().split())) for _ in range(length)]

result = sorted(coords)
for coord in result:
    print(coord[0], coord[1])
